pkg/conduit/config.go

```diff
@@ -59,6 +59,8 @@ type Config struct {
 
 	Pipelines struct {
 		Path string
+		// todo there's probably a better name
+		StrictMode bool
```

Comment:
I like `exitOnError` more tbh, i think `failFast` works too

Reply:
`exit-on-error` got most votes. I put `StrictMode` because it started to appear in the last call we had, so I thought it would be good to discuss it. Thanks for the inputs!